packages/fxa-auth-server/lib/senders/emails/layouts/subscription/index.scss

```diff
@@ -47,6 +47,10 @@
   padding: global.$s-5 global.$s-0 !important;
   height: 56px !important;
   display: block !important;
+
+  a {
+    font-size: 13px !important;
```

Comment:
Is it possible to `@extend .text-xs` or `@extend .text-sm` here instead? Not sure where `13px` is coming from since nothing else in our emails should use that font-size 🤔